configs/components/windows_puppet.json

```diff
@@ -1,4 +0,0 @@
-{
-  "url": "git://github.delivery.puppetlabs.net/puppetlabs-puppet_for_the_win.git",
-  "ref": "origin/master"
-}
```

Comment:
Can we delete [`windows_ruby.json`](https://github.com/puppetlabs/puppet-agent/blob/2.0.x/configs/components/windows_ruby.json) also and references to it?